pytest_tests/lib/helpers/acl.py

```diff
@@ -42,6 +43,26 @@ class EACLRole(Enum):
     SYSTEM = "system"
 
 
+class EACLRoleExtendedType(Enum):
+    PUBKEY = "pubkey"
+    ADDRESS = "address"
+
+    def get_value(self, wallet_path: str, wallet_password: str) -> str:
```

Comment:
Why `wallet_password` here? Technically you don't need it, public key is _public_, address is available as well.

Reply:
It goes down to the APIs of the neo3 python package which requires a wallet password to be specified to even parse a wallet json file. I can implement my own methods to work with it, but not sure if it is worth it.